spark/v3.3/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestAddFilesProcedure.java

```diff
@@ -974,6 +974,55 @@ public void testPartitionedImportFromEmptyPartitionDoesNotThrow() {
         sql("SELECT * FROM %s ORDER BY id", tableName));
   }
 
+  @Test
+  public void testSkipOnError() throws IOException {
+    createUnpartitionedFileTable("parquet");
+
+    String createIceberg =
+        "CREATE TABLE %s (id Integer, name String, dept String, subdept String) USING iceberg";
+
+    sql(createIceberg, tableName);
+
+    // Create an empty(considered corrupted) file.
+    Assert.assertTrue(new File(fileTableDir + File.separator + "corrupt.parquet").createNewFile());
+
+    AssertHelpers.assertThrows(
```

Comment:
just an FYI that we're trying to move people away from using `AssertHelpers` (https://github.com/apache/iceberg/pull/6977)

Reply:
Thanks. I've made a replacement.